daffodil-core/src/test/resources/test/standard-xsd/bmp.xsd

```diff
@@ -0,0 +1,656 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
We generally avoid putting large schemas for file types inside the daffodil repo. Instead, we create smaller schemas that test the cases we are concerned about.

Reply:
Yes, I noticed that - this paradigm makes more sense than what I had.  In the JUnit-based test class in the latest revision, the schemas are in the code and passed to the helper "dfdlTestSchema" I saw being used in the other test cases.  Let me know if you think the one I made for SimpleTypes is too long for a Unit Test environment; I could always break up each SimpleType into its own test